flight/Libraries/math/pid.c

```diff
@@ -199,6 +230,67 @@ void pid_configure(struct pid *pid, float p, float i, float d, float iLim)
 }
 
 /**
+ * Assigns a pid_deadband struct to the PID controller.
+ * @param[out] pid The PID structure to configure
+ * @param[in] deadband The external deadband structure to assign
+ */
+void pid_assign_deadband(struct pid *pid, struct pid_deadband *db)
+{
```

Comment:
Can this be done dynamically (and lazily) in pid_configure_deadband?  (e.g. have it take a struct pid * instead), and malloc the structure if it is not present.  Then at least F1 would only pay the tax for this feature if deadbands are configured.

(we're already paying the cost of a pointer-- might as well malloc)